src/kernel-install/90-loaderentry.install.in

```diff
@@ -66,7 +66,10 @@ elif [ -f /usr/lib/os-release ]; then
     . /usr/lib/os-release
 fi
 
-[ -n "$PRETTY_NAME" ] || PRETTY_NAME="Linux $KERNEL_VERSION"
+[ -n "$PRETTY_NAME" ] || PRETTY_NAME="Linux"
+
+# Add Kernel Version
+PRETTY_NAME="$PRETTY_NAME $KERNEL_VERSION"
```

Comment:
Not a matter, but I'd like to append the kernel version when writing `title: ` field below, and keep `$PRETTY_NAME` as loaded from os-release file (or `Linux` if not set).

Reply:
Solved :)
Thanks @yuwata